src/tscpp/util/ts_diags.cc

```diff
@@ -0,0 +1,33 @@
+/** @file Support for common diagnostics between core, plugins, and libswoc.
```

Comment:
Aren't we missing this file into the build? (Makefile.am)

Reply:
Those changes are incidental and one reason it's still a draft. I made a 1.4.0 release of libswoc and I'll set up a PR to bring that in to 10-Dev, which should remove all of the libswoc changes in this PR. Let me check on "ts_diags.cc" - I'm working on yet another PR to make just the `Errata` changes and that's part of it.